submissions/HelenGreent/task_html_css_popup/index.css

```diff
@@ -0,0 +1,247 @@
+}
+
+body {
+  font-family: "Work Sans", sans-serif;
+  background-color: var(--main-color);
+}
+
+.header {
+  padding: 0.6rem 0;
+  background-color: var(--bg-color);
+  box-shadow: var(--box-shadow);
+  position: relative;
+}
+
+.main__nav {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  padding: 0 1rem;
+}
+
+.main__nav-item {
+  padding: 0rem 1rem;
+  font-size: 1.2rem;
+  font-weight: 500;
+  color: var(--text-color);
+  text-decoration: none;
+  outline: none;
+  border: 1px solid transparent;
+}
+
+.main__nav-item > img {
+  height: 1rem;
+  padding-right: 0.2rem;
+}
+
+.main__nav-item:focus {
+  border: var(--border-focus);
+}
+
+.main__nav-item:hover {
+  border: var(--border-hover);
+}
+
+.nav {
+  padding: 0rem 1rem;
+}
+
+.nav__list {
+  list-style: none;
+}
+
+.nav__link {
+  text-decoration: none;
+  outline: none;
+}
+
+.nav__link:focus {
+  border: var(--border-focus);
+}
+
+.nav__link:hover {
+  border: var(--border-hover);
+}
+
+.popup {
+  position: relative;
+}
+
+.popup__toggler {
+  padding: 0.25rem;
+  display: flex;
+  align-items: center;
+}
+
+.popup__control {
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  appearance: none;
+  outline: none;
+  cursor: pointer;
+  border: 2px solid transparent;
+}
+
+.popup__control:focus {
+  border: var(--border-focus);
+}
+
+.popup__control:hover {
+  border: var(--border-hover);
+}
+
+.popup__control:checked ~ .popup__content {
+  display: block;
+}
+
+.popup__content {
+  display: none;
+  position: absolute;
+  top: 3.5rem;
+  right: -5.5rem;
+  min-width: 300px;
+  max-height: 500px;
+  padding: 0.6rem;
+  overflow-y: auto;
+  overflow-x: hidden;
+  background-color: var(--bg-color);
+  box-shadow: var(--box-shadow);
+}
+
+.popup__content::before {
+  content: "";
+  position: fixed;
+  top: 2.25rem;
+  right: 10rem;
+  transform: translateX(-50%);
+  border: 15px solid;
+  border-color: transparent transparent var(--bg-color) transparent;
+  z-index: 1;
+}
+
+.popup__list {
+  display: flex;
+  justify-content: space-around;
+  flex-wrap: wrap;
+  list-style: none;
+}
+
+.popup__list-item {
+  display: flex;
+  flex-flow: column wrap;
+  justify-content: space-around;
+  width: 30%;
+  height: 7rem;
+  text-align: center;
+}
+
+.popup__link {
+  padding: 0.3rem;
+  text-decoration: none;
+  outline: none;
+  color: inherit;
+  border: 1px solid transparent;
+}
+
+.popup__link-title {
+  display: flex;
+  flex-flow: column wrap;
+  justify-content: flex-end;
+  height: 3.5rem;
+  padding: 1.5rem 0.1rem;
+  font-size: 0.9rem;
+  text-align: center;
+}
+
+.popup__link:focus {
+  border: var(--border-focus);
+}
+
+.popup__link:hover {
+  border: var(--border-hover);
+}
+
+.popup__link-icon {
+  max-width: 3rem;
+  height: 2rem;
+  object-fit: contain;
+}
+
+.add-popup {
+  position: relative;
+}
+
+.add-popup__toggler {
+  width: 100%;
+  text-align: center;
+  padding: 0.6rem;
+}
+
+.add-popup__list-item {
```

Comment:
I can't find where you use this class in HTML. Please, don't hurry, take your time and review your code. Sometimes you need just to take a one-day rest and make a self-code-review to find some gaps. I know it from my own experience ;)